assets/js/builder/Views/VideoExplainer.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Sidebar Utilities View
+ *
+ * @since [version]
+ * @version [version]
+ */
+define( [], function() {
+	return Backbone.View.extend( {
+
+		/**
+		 * HTML element selector
+		 *
+		 * @type {string}
+		 */
+		el: '#llms-video-explainer',
+
+		/**
+		 * Wrapper Tag name
+		 *
+		 * @type {string}
+		 */
+		tagName: 'div',
+
+		/**
+		 * Events
+		 *
+		 * @type {Object}
+		 */
+		events: {
+			'click .llms-video-explainer-trigger': 'openPopup',
+			'click .llms-video-explainer-close': 'closePopup',
+			'click .llms-video-explainer-wrapper': 'closePopup',
+		},
+
+		/**
+		 * Youtube video url
+		 *
+		 * @type {string}
+		 */
+		youtubeUrl: 'https://www.youtube.com/embed/kMd37cOsPIg',
+
+		/**
+		 * Get the underscore template
+		 *
+		 * @type {Function}
+		 */
+		template: wp.template( 'llms-video-explainer-template' ),
+
+		/**
+		 * Initialization callback func (renders the element on screen)
+		 *
+		 * @since [version]
+		 * @version [version]
+		 * @type {Function}
+		 * @return {void}
```

Comment:
```suggestion
		 * @since [version]
		 * 
		 * @return {void}
```
no `@version` for the methods/functions, I don't think we need the `@type` for methods as well.
Same thing for the `render` method below.